pkg/proxy/accesslog/log.go

```diff
@@ -127,3 +204,102 @@ func Log(l *LogRecord, typ FlowType, verdict FlowVerdict, code int) {
 		log.WithError(err).WithField(FieldFilePath, logPath).Warn("Error encountered while flushing to access log")
 	}
 }
+
+// fillIdentity resolves the labels of the specified identity if known
+// locally and fills in the following info member fields:
+//  - info.Identity
+//  - info.Labels
+//  - info.LabelsSHA256
+func (info *EndpointInfo) fillIdentity(id policy.NumericIdentity) {
+	info.Identity = uint64(id)
+
+	if c := policy.GetConsumableCache().Lookup(id); c != nil {
+		if c.Labels != nil {
+			info.Labels = c.Labels.Labels.GetModel()
+			info.LabelsSHA256 = c.Labels.GetLabelsSHA256()
+		}
+	}
+}
+
+// egressDestinationInfo returns the destination EndpointInfo for a flow
+// leaving the proxy at egress.
+func (info *EndpointInfo) egressDestinationInfo(ipstr string) {
+	ip := net.ParseIP(ipstr)
```

Comment:
I've seen this code in PR #1870 @manalibhutiyani or @jrajahalme is this duplicated or related?

Reply:
We can rebase 1870 on top of this, 1870 is currently duplicating the http code for kafka

Reply:
I have removed the duplication actually.